### PR TITLE
Fix issue1

### PR DESCRIPTION
**Pull request by vutkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,15 +61,15 @@ Example Playbook
 - name: Converge
   hosts: all
   roles:
-    - role: lean_delivery.docker
+    - role: ansible-role-docker
```
Comment (tgadiev) [inline README.md]: Why changing the name?

File: README.md [hunk 2]
```diff
@@ -78,15 +79,15 @@ Example Playbook
...
```
Comment (tgadiev) [inline README.md]: ???

File: tasks/install-docker.yml [hunk 3]
```diff
@@ -5,6 +5,8 @@
     name: "{{ docker_package }}"
     state: present
   become: True
+  register: __installed_docker
+  until: __installed_docker is succeeded
   notify: restart docker
 
 - block:
```
Comment (tgadiev) [inline tasks/install-docker.yml]: Why do you need block for single task?

**Review by tgadiev — COMMENTED** [review]
Please explain some items

File: README.md (hunk 1 above)
Comment (vutkin) [reply README.md]: Fixed

File: tasks/install-docker.yml (hunk 3 above)
Comment (vutkin) [reply tasks/install-docker.yml]: Fixed

File: README.md (hunk 2 above)
Comment (vutkin) [reply README.md]: Fixed

**Pull request merged**